### PR TITLE
Remove inapropriated error when a Folder is replicated from the Cozy.

### PR DESCRIPTION
**Pull request by jacquarg**

### CODE REVIEW

File: www-src/app/replicator/change/change_dispatcher.coffee [hunk 1]
```diff
@@ -37,8 +36,9 @@ module.exports = class ChangeDispatcher
     dispatch: (doc, callback = ->) ->
         log.info "dispatch #{doc.docType}"
 
-        if @isDispatched(doc) and @changeHandlers[doc.docType.toLowerCase()]
```
Comment (aenario) [inline www-src/app/replicator/change/change_dispatcher.coffee]: isDispatched is now dead code, remove it.

Comment (jacquarg) [inline www-src/app/replicator/change/change_dispatcher.coffee]: No ! It's used here https://github.com/jacquarg/cozy-mobile/blob/development/www-src/app/replicator/replication_launcher.coffee#L62

Comment (aenario) [inline www-src/app/replicator/change/change_dispatcher.coffee]: My bad.

Comment (jacquarg) [inline www-src/app/replicator/change/change_dispatcher.coffee]: No worries.
This file is almost brand new and contains already 'historical' method, and trace of fight between devs : )

**Pull request merged**